### ISSUE TITLE
App ain't working after reinstalling

### ISSUE THREAD

**Issue by kasrabo** [NotCoffee418/TrinityCreator#61]
hey, so installed the app ( Great app btw ) every thing was OK but then reinstalled it and it's not opening now.  I have noticed that when I open the app instantly it get's suspended so I disabled the anti virus (Windows defender) but still nothing.

**Comment by illfated** [comment]
Maybe there is something in the app that triggers Windows Defender to block the app on your computer during install. You may want to disable Windows Defender, both while uninstalling and reinstalling, before trying to run the app again.

**Comment by NotCoffee418** [comment]
I'm thinking more along the lines of old settings crashing the app.
For example, database or DBC path attempting to load on startup but the DB/dbc has been moved or deleted.
Either way, needs proper error handling.

If anyone has this issue before i fix it do the following:
1. Open task manager > Processes > kill TrinityCreator.exe processes. (or just reboot)
2. Go to C:\Users\YOURUSERNAMEHERE\AppData\Local (Appdata is hidden, use Win+R or enable view hidden folders to access it)
And delete folders named Nadromar or NotCoffee418 if they exist. This will wipe the application settings for TrinityCreator.

**Comment by illfated** [comment]
Or even easier, to help users skip looking up the exact spelling of their username:

1. Windows Key + R
2. copy & paste `C:\Users\%UserName%\AppData\Local\`
3. Click [OK] / press Enter.

**Comment by NotCoffee418** [comment]
Gotta remember that one... I usually just do %appdata% and navigate manually.

**Comment by illfated** [comment]
useful Windows CMD environment variables:
```
C:\>set

ALLUSERSPROFILE=C:\ProgramData
APPDATA=C:\Users\%USERNAME%\AppData\Roaming
ComSpec=C:\WINDOWS\system32\cmd.exe
DriverData=C:\Windows\System32\Drivers\DriverData
HOME=%HOMEDRIVE%%HOMEPATH%
HOMEDRIVE=C:
HOMEPATH=\Users\%USERNAME%
LOCALAPPDATA=C:\Users\%USERNAME%\AppData\Local
ProgramData=C:\ProgramData
ProgramFiles=C:\Program Files
ProgramFiles(x86)=C:\Program Files (x86)
ProgramW6432=C:\Program Files
PROMPT=$P$G
PUBLIC=C:\Users\Public
SystemDrive=C:
SystemRoot=C:\WINDOWS
TEMP=C:\Temp
TMP=C:\TMP
USERNAME=
USERPROFILE=C:\Users\%USERNAME%
windir=C:\WINDOWS

C:\>_
```
... so for the above, `LOCALAPPDATA=C:\Users\%USERNAME%\AppData\Local` is most useful here.

**Comment by NotCoffee418** [comment]
Should be fixed in [v1.1.0.103](https://github.com/NotCoffee418/TrinityCreator/releases/tag/v1.1.0).
Caused by CefSharp missing some locale or other file.

If anyone who had this issue recently can verify, please let me know.
Otherwise i'll close this issue in a while.
